### PR TITLE
Fix broken links in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by hakeemullahjan**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ When developing a new feature, please put it behind a feature flag. Because this
 1. Fork tahowallet/tally-extension
```
Comment (hyphenized) [inline CONTRIBUTING.md:32]:
```suggestion
1. Fork tahowallet/extension
```

**Review by hyphenized — CHANGES_REQUESTED** [review]
Thanks for the help! Noticed a small correction that we could include in this PR, could you fix that too?

File: CONTRIBUTING.md (hunk 1 above)
Comment (hakeemullahjan) [reply CONTRIBUTING.md:32]: Yeah, sure!

**Review by hyphenized — APPROVED** [review]
Looks great, thanks!

**Pull request merged**